src/main/java/seedu/souschef/logic/commands/AddCommand.java

```diff
@@ -54,7 +83,18 @@ public AddCommand(Model model, T toAdd) {
     public CommandResult execute(CommandHistory history) {
         model.add(toAdd);
         model.commitAppContent();
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+        String context = history.getContext();
+        //switch case to handle the add for the different context
+        switch(context) {
```

Comment:
For this, we can just insert "return new CommandResult(String.format(MESSAGE_SUCCESS, context, toAdd));" after setting setting the context. Please remove related implementations! Also context of recipe is "Recipe".